docs/dev/api/insights.md

```diff
@@ -1111,32 +1217,128 @@ curl -u "$SAUCE_USERNAME:$SAUCE_ACCESS_KEY" --location \
 
 ---
 
-### Get Activity for Users
+### Get Filter Items from All Sources
 
-<details><summary><span className="api get">GET</span> <code>/rest/v1/users_activity</code></summary>
+<details><summary><span className="api get">GET</span> <code>/insights/v2/filters</code></summary>
 <p/>
 
-Return daily statistics about test results and concurrency usage.
+Return data, from all sources, for all tests that match the request criteria.
 
 #### Parameters
 
 <table id="table-api">
   <tbody>
     <tr>
-     <td><code>users</code></td>
-       <td><p><small>| QUERY | OPTIONAL | ARRAY of STRINGS |</small></p><p>Limit results to specific users.</p></td>
+     <td><code>org_id</code></td>
+       <td><p><small>| QUERY| REQUIRED | STRING |</small></p><p>Return results only for the specified <code>org_id</code>.</p></td>
     </tr>
   </tbody>
   <tbody>
     <tr>
-     <td><code>since</code></td>
-       <td><p><small>| QUERY | OPTIONAL | DATE |</small></p><p>Return only jobs ran from the provided Unix timestamp on.</p></td>
+     <td><code>status</code></td>
+     <td><p><small>| QUERY | OPTIONAL | ARRAY of STRINGS |</small></p><p>Limit results to only those with a specified status. Supported values are:</p><p>
+     <ul>
+      <li><code>passed</code></li>
+      <li><code>error</code></li>
+      <li><code>failed</code></li>
+      <li><code>complete</code></li>
+    </ul></p>Default value is: <code>["error", "failed", "passed", "complete"]</code>
```

Comment:
@LoryApiFortress All values are default? The description of the same parameter for the [Analytics > Get Test Results](https://docs.saucelabs.com/dev/api/insights/#get-test-results) endpoint doesn't include a default.